Gemfile

```diff
@@ -63,13 +63,14 @@ end
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
-  gem 'rspec-rails'
-  gem 'shoulda-matchers'
-  gem 'guard-rspec'
-  gem 'guard-rubocop'
+  gem 'capybara'
+  gem 'dotenv-rails'
```

Comment:
Try to avoid moving things around unnecessarily. I can't tell what you added easily here. Is it just capybara?